### DISCUSSION TITLE
GitHub Widget Not Working on Poco X6 Pro

### DISCUSSION THREAD

**Discussion by Tahermaxse** [community/community#141508] [Mobile]
I am writing to report a persistent issue with the GitHub widget on my Poco X6 Pro device. Despite having the latest version of the GitHub app installed and a stable internet connection, the widget consistently fails to load correctly. It remains blank or displays an error message, preventing me from accessing recent activity or notifications directly from my home screen.

I have tried troubleshooting by restarting my device, clearing the GitHub app cache and data, and even reinstalling the app. Unfortunately, none of these steps have resolved the problem.

I would appreciate it if you could investigate this issue and provide guidance on how to resolve it. Perhaps there's a specific compatibility issue or a known bug that needs addressing.

Thank you for your time and attention to this matter.

**Comment by RaHorEl** [comment]
Working on your soul being taken
